### PR TITLE
EVAC+ workflow

### PR DESCRIPTION
**Pull request by pjsjongsung**
This adds a workflow for EVAC+ with a test for it.

### CODE REVIEW

File: dipy/workflows/segment.py [hunk 1]
```diff
@@ -334,3 +335,49 @@ def run(self, streamline_files, labels_files,
...
+
+class EVACPlusFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'evacplus'
+
+    def run(self, input_files, save_masked=False, out_dir='',
+            out_mask='brain_mask.nii.gz', out_masked='dwi_masked.nii.gz'):
+        """ Extract brain using EVAC+
+        Parameters
```
Comment (skoudoro) [inline dipy/workflows/segment.py:347]: missing empty line above

**Review by skoudoro — APPROVED** [review]
Looks good to me, Thanks @pjsjongsung!

See below a small comment but this is ready to go. I am waiting for your update and the CI's to finish

File: dipy/workflows/tests/test_segment.py [hunk 2]
```diff
@@ -102,3 +103,30 @@ def test_recobundles_flow():
         bmd_value = BMD.distance(x0.tolist())
 
         npt.assert_equal(bmd_value < 1, True)
+
+def test_evac_plus_flow():
```
Comment (skoudoro) [inline dipy/workflows/tests/test_segment.py:107]: this tests is missing a `skipif not have_tf`

**Comment by pjsjongsung** [comment]
@skoudoro the workflow file is in bin but does not seem to work when I try dipy_evac_plus. Is there another file I have to change?

**Comment by skoudoro** [comment]
No, did you check if it is executable ?  (Chmod +x)

**Comment by pjsjongsung** [comment]
I can run it within the bin directory but cannot seem to access it anywhere like dipy_horizon

**Comment by pjsjongsung** [comment]
@skoudoro should be all done!

File: dipy/workflows/segment.py [hunk 3]
```diff
@@ -358,14 +359,27 @@ def run(self, input_files, save_masked=False, out_dir='',
...
             Name of the masked volume to be saved.
+
+        References
+        ----------
+        ..  [Park2022] Park, J.S., Fadnavis, S., & Garyfallidis, E. (2022).
+        EVAC+: Multi-scale V-net with Deep Feature
+        CRF Layers for Brain Extraction.  
+
         """
+        if os.path.exists(out_mask) or os.path.exists(out_masked):
```
Comment (skoudoro) [inline dipy/workflows/segment.py:370]: it should not be done here. it should be done in the for loop with `mask_out_path` and `masked_out_path`

**Review by skoudoro — APPROVED** [review]
LGTM, Thanks @pjsjongsung, merging

**Pull request merged**